ArcGISRuntimeSDKQt_CppSamples/Maps/MapReferenceScale/README.md

```diff
@@ -5,15 +5,15 @@ Set the map's reference scale and which feature layers should honor the referenc
 
 ## Use case
 
-Setting a reference scale on an `Map` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `MapView`.
+Setting a reference scale on a `Map` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `MapView`.
 
 Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `Map`. 
 
 ## How to use the sample
 
-* Use the drop box at the top to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
-* Click the button to set the map scale to the reference scale.
-* Use the menu checkboxes in the layer menu to set which feature layers should honor the reference scale.
+1. Use the drop box at the top to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
+1. Click the button to set the map scale to the reference scale.
+1. Use the menu checkboxes in the layer menu to set which feature layers should honor the reference scale.
```

Comment:
This looks correct in a markdown viewer, but just looking at the md file it won't. Would you be opposed to numbering these as just 1, 2, 3 ?

Reply:
Consider it done.